### PR TITLE
#98 Admin check who real come to the event

### LINKED ISSUE #98: Event visit confirmation by admin
Admin check who really visited the event

### PR DESCRIPTION
**Pull request by sveredyuk**
Fixed #98

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -4,14 +4,15 @@ class Event < ApplicationRecord
 
   PLANNED       = :planned
   REGISTRATION  = :registration
+  LIVE          = :live
```
Comment (denys-medynskyi) [inline app/models/event.rb:7]: do we need `live` status, because we have start/end at fields ?

**Review by denys-medynskyi — APPROVED** [review]

File: app/models/event.rb (hunk 1 above)
Comment (sveredyuk) [reply app/models/event.rb:7]: First I also thought that.. but manual control is a little better...

Comment (denys-medynskyi) [reply app/models/event.rb:7]: ok

Comment (sveredyuk) [reply app/models/event.rb:7]: And also will be much easier build query on top of status instead time period...

**Pull request merged**